apps/test/solutions/applab/ec_screens.js

```diff
@@ -158,7 +152,8 @@ module.exports = {
 
         ReactTestUtils.Simulate.click($("#design-properties button").eq(-1)[0]);
```

Comment:
If you don't mind modifying some of the existing code, please add a comment that says "Click the delete button" here.